tests/unit/test_ticket_categorization.py

```
@@ -0,0 +1,88 @@
+
+    async def test_router_validates_at_least_one_field_provided(self):
+        """Error: PATCH payload with neither category nor priority → ValidationException."""
+        from api.routers.tickets import CategorizationFeedbackPayload
+
+        payload = CategorizationFeedbackPayload()
+        assert payload.category is None and payload.priority is None
+        # Simulate router validation logic
+        if payload.category is None and payload.priority is None:
+            with pytest.raises(ValidationException) as exc_info:
+                raise ValidationException("At least one of category or priority must be provided")
+        assert "At least one of category or priority" in str(exc_info.value)
```

Pylint (4.0.5)

[error] Possibly using variable 'exc_info' before assignment

(E0606)